docs/TESTS.md

```diff
@@ -1,112 +1,65 @@
-## Xcode Tutorial
+## Testing Tutorial
 
-This guide explains how to create and configure a Swift-based Xcode project for Exercism.
+This guide explains how to run unit tests for Exercism Swift exercises, and optionally how to use Xcode for development.
 
-Apple does not support user-defined project templates in Xcode, and the default templates are more elaborate than those needed for Exercism. This guide bridges that gap.
+Exercism uses [Swift Package Manager](https://github.com/apple/swift-package-manager/tree/master/Documentation) to package files and tests for Swift.
```

Comment:
I should have added, I'm coming at this purely from the Mac, and only recently began using SPM at all. I just worked though an example and think that tests.md is excellent, and the issues I noted above remain but are very simple (brief) to address. I also saw a few other sentences that might be out of place.